Web/starter_project_barbies/styles/blog/blog-detail.module.css

```diff
@@ -0,0 +1,10 @@
+@import url('https://fonts.googleapis.com/css2?family=IM+Fell+French+Canon&family=Montserrat:wght@200&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
+
+.cannon {
+  font-family: 'IM FELL French Canon';
+}
+
+.monteserrat {
+  font-family: 'Montserrat';
+}
```

Comment:
There's another way to set the font-family. You can define it on the tailwind.config file instead of doing it on the css file.

Reply:
Yeah I guess you're right. Will change it.